app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -37,8 +37,11 @@ def award_reputation_to_reviewers
         reviews = octokit_client.pull_request_reviews(repo, pr_id)
         reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }
 
-        reviewers = ::User.where(handle: reviewer_usernames)
+        reviewers = ::User.where(github_username: reviewer_usernames)
         reviewers.find_each do |reviewer|
+          # Don't award reputation for reviews by the PR author
+          next if reviewer.github_username == github_username
```

Comment:
I could also do:

```ruby
reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }
reviewer_usernames.delete(github_username)  # Don't award reputation for reviews by the PR author
```

and remove the check here. Somehow I felt that this read a bit better, although it does come at the cost of returning one user that we won't be used (although it _will_ then be properly logged in the error log below. Let me know what you prefer.